File: elza/elza-react/src/api/ExtSyncsQueueResultListVO.ts
/**
 * výsledek hledání
 * @export
 * @interface ExtSyncsQueueResultListVO
 */
import {ExtSyncsQueueItemVO} from "./ExtSyncsQueueItemVO";

export interface ExtSyncsQueueResultListVO {
    /**
     * celkový počet
     * @type {number}
     * @memberof ExtSyncsQueueResultListVO
     */
    total: number;
    /**
     * výsledky
     * @type {Array<ExtSyncsQueueItemVO>}
     * @memberof ExtSyncsQueueResultListVO
     */
    data: Array<ExtSyncsQueueItemVO>;
}

File: elza/elza-core/src/main/java/cz/tacr/elza/controller/vo/ExtSyncsQueueResultListVO.java
package cz.tacr.elza.controller.vo;

import java.util.ArrayList;
import java.util.List;

public class ExtSyncsQueueResultListVO {

    private Integer total = 0;

    private List<ExtSyncsQueueItemVO> data = new ArrayList<>();

    public Integer getTotal() {
        return total;
    }

    public void setTotal(Integer total) {
        this.total = total;
    }

    public List<ExtSyncsQueueItemVO> getData() {
        return data;
    }

    public void setData(List<ExtSyncsQueueItemVO> data) {
        this.data = data;
    }
}
